common-crafting.lic

```diff
@@ -460,4 +463,27 @@ module DRCC
     end
   end
 
+  def check_for_existing_sigil?(sigil, stock_number, quantity, bag, belt, info)
+    merged = Regexp.union($PRIMARY_SIGILS_PATTERN, $SECONDARY_SIGILS_PATTERN)
+
+    tmp_count = 0
+    more = 0
+    tmp_count = DRCI.count_items_in_container("#{sigil} sigil-scroll", bag)
+
+    if tmp_count >= quantity
+      return true
+    else
+      if merged.match?("#{sigil} sigil")
+        more = quantity - tmp_count
+        # Found a weird challenge that made the temp_part_count equal 1 even though no "sigil" was in container
+        more += 1 if reget(3, "but there is nothing in there like that")
+        DRCC.order_enchant(info['stock-room'], more, stock_number, bag, belt)
+        return true
+      else
+        DRC.message("Not enough #{sigil} sigil-scroll(s). You can purchase or harvest #{more} more. We recommend using our sigilhunter script. Run ;sigilhunter help for more information.")
```

Comment:
This is likely deprecated, since they're all purchaseable.

Reply:
Except for guild specific and any new ones they want to throw in. I would like to go proactive on this instead of reactive.